### PR TITLE
feat: upgraded to 2.26 geoserver version

### PR DESCRIPTION
**Pull request by CptSchnitz**

### CODE REVIEW

**Review by NivGreenstein — APPROVED** [review]

File: Dockerfile [hunk 1]
```diff
@@ -7,6 +7,8 @@ ARG LOG4J_VERSION
 ARG JMX_PROMETHEUS_VERSION
 
 ENV OTEL_SERVICE_NAME=geoserver
+ENV RUN_AS_ROOT=true
+ENV OTEL_LOGS_EXPORTER=none
```
Comment (syncush) [inline Dockerfile:11]: what are the options ? needs docs

File: index.mjs [hunk 2]
```diff
@@ -1,13 +1,13 @@
 #!/usr/bin/env zx
```
Comment (syncush) [inline index.mjs:1]: did you run it from end to end to see if it works?

**Review by syncush — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (CptSchnitz) [reply Dockerfile:11]: both of them are internal options for geoserver and the otel instrumentation, they should not be changed.

File: index.mjs (hunk 2 above)
Comment (CptSchnitz) [reply index.mjs:1]: define end to end. we did run and get the UI as far as i remember, but didnt connect to pg

**Pull request merged**